app/views/abuse_reports/new.html.erb

```diff
@@ -28,7 +28,7 @@
             (link_to ts("DMCA Policy"), dmca_path)).html_safe %></li>
   </ul>
   <p><%= ts("<strong>We can answer Abuse reports in %{list}.</strong> Please allow for an additional delay for responses
-            in any language other than English.", list: @abuse_languages.map(&:name).to_sentence).html_safe %></p>
+            in any language other than English.", list: @abuse_languages.pluck(:name).to_sentence).html_safe %></p>
```

Comment:
I'm not 100% certain, but I think that the call to `f.collection_select` on line 50 below will force `@abuse_languages` to be loaded:
https://github.com/otwcode/otwarchive/blob/df6ce85db5cb1a857186215e43cd0cafba3234a2/app/views/abuse_reports/new.html.erb#L50
I think this is a case where the switch to pluck just results in an extra database query.